plugins/BEdita/Core/src/Model/Action/ListEntitiesAction.php

```diff
@@ -138,6 +140,13 @@ protected function buildFilter(Query $query, array $filter)
 
                 continue;
             }
+
+            // No suitable filter was found
+            //$this->log('Filter not found ' . $key, 'error');
+            throw new BadRequestException([
```

Comment:
I'm not quite sure it is a good idea to raise a `BadRequestException` from within the model layer.

Perhaps I'd prefer a `new \LogicException('...', 400)`. Or, even better, create a custom `BadFilterCallException` that has `400` as its default code.